source/includes/interact-data/transaction.rb

```diff
@@ -0,0 +1,132 @@
+# start-example-models
+class Book
+    include Mongoid::Document
+
+    field :title, type: String
+    field :author, type: String
+    field :length, type: Integer
+end
+
+class Film
+    include Mongoid::Document
+
+    field :title, type: String
+    field :year, type: Integer
+end
+# end-example-models
+
+# start-txn-operations
+# Starts a transaction from the model class
+Book.transaction do
+    # Saves new Book and Film instances to MongoDB
+    Book.create(title: 'Covert Joy', author: 'Clarice Lispector')
+    Film.create(title: 'Nostalgia', year: 1983)
+end
+
+# Starts a transaction from an instance of Book
+book = Book.create(title: 'Sula', author: 'Toni Morrison')
+book.transaction do
+  # Saves a new field value to the Book instance
+  book.length = 192
+  book.save!
+end
+
+# Starts a transaction from the Mongoid instance
+Mongoid.transaction do
+  # Deletes the Book instance in MongoDB
+  book.destroy
+end
+# end-txn-operations
+
+# start-different-clients
+# Defines a class by using the :default client
+class Post
+  include Mongoid::Document
+end
+  
+# Defines a class by using the :encrypted_client
+class User
+  include Mongoid::Document
+
+  store_in client: :encrypted_client
+end
+
+# Starts a transaction on the :encrypted_client
+User.transaction do
+  # Uses the same client, so the operation is in the transaction
+  User.create!
+  # Uses a different client, so it is *not* in the transaction
+  Post.create!
+end
+# end-different-clients
+
+# start-lower-lvl-api
+# Starts a session from the model class
+Book.with_session do |session|
+  # Starts the transaction in the session
+  session.start_transaction
+end
+
+book = Book.new
+# Starts a session from an instance of Book
+book.with_session do |session|
+  # Starts the transaction in the session
+  session.start_transaction
```

Comment:
I am not sure whether this and following two examples make sense. We should call `session.start_transaction` and `session.commit_transaction` or `session.abort_transaction` within the same `with_session` block. Maybe we should combine there examples into one.

Reply:
Discussed the updated code example over slack